project/sites/nicybersecuritycentre/themes/nicybersecuritycentre_theme/templates/layout/page--events.html.twig

```diff
@@ -0,0 +1,256 @@
+ *   associated with the page and the node ID is the second argument in the
+ *   page's path (e.g. node/12345 and node/12345/revisions, but not
+ *   comment/reply/12345).
+ *
+ * Regions:
+ * - page.header: Items for the header region.
+ * - page.search: Items for site search region.
+ * - page.primary_menu: Items for the primary menu region.
+ * - page.slideshow: Items for the slideshow region on the front page.
+ * - page.front_welcome: Items for the front page welcome region.
+ * - page.top_first: Items for the first top region.
+ * - page.top_second: Items for the second top region.
+ * - page.top_third: Items for the third top region.
+ * - page.breadcrumb: Items for the breadcrumb region.
+ * - page.search_facets: Search facet region for search pages.
+ * - page.content: The main content of the current page.
+ * - page.sidebar_first: Items for the first sidebar.
+ * - page.sidebar_second: Items for the second sidebar.
+ * - page.bottom_widget_1: Items for the first bottom widget region on the front page.
+ * - page.bottom_widget_2: Items for the second bottom widget region on the front page.
+ * - page.bottom_widget_3: Items for the third bottom widget region on the front page.
+ * - page.footer_first: Items for the first footer region.
+ * - page.footer_second: Items for the second footer region.
+ * - page.footer_third: Items for the third footer region.
+ * - page.footer_fourth: Items for the fourth footer region.
+ * - page.footer: Items for the bottom footer region.
+ *
+ * - basic_page_structure: There is a checkbox in the theme settings UI
+ *   to check if a basic page is used in the homepage structure. This template
+ *   is set up for a homepage that doesn't use a basic page in it's structure.
+ *
+ * @see template_preprocess_page()
+ * @see html.html.twig
+ */
+#}
+{% set hero_attributes = create_attribute() %}
+{% set hero_classes = [
+  'top_right',
+  page.primary_menu ? 'with-menu' : 'no-menu',
+  page.search ? 'with-search' : 'no-search',
+] %}
+
+<div id="header_wrapper">
+  <div id="inner_header_wrapper">
+    <header id="header">
+      <div class="top_left">
+        {{ page.header }}
+      </div>
+      <div{{ hero_attributes.addClass(hero_classes).setAttribute('id', 'headerHero') }}>
+        {{ page.search }}
+      </div>
+      <div class="clear"></div>
+    </header>
+  </div>
+  {% if page.primary_menu %}
+    {{ page.primary_menu }}
+  {% endif %}
+  {% if page.header_block %}
+    {{ page.header_block }}
+  {% endif %}
+</div>
+
+{% if is_front %}
+  {{ page.slideshow }}
+{# If the homepage has no basic page then make the content the Main section and add the h1. #}
+{% if basic_page_structure is empty %}
+<main id="content">
+  {% if drupal_config('system.site', 'frontpage_title') %}
+    <h1 class="{{ drupal_config('system.site', 'frontpage_show_title') ? '' : 'visually-hidden' }}">
+      {{ drupal_config('system.site', 'frontpage_title') }}
+    </h1>
+  {% endif %}
+  {% endif %}
```

Comment:
Slight nitpick here. It'd be easier to read if this was aligned to its matching 'if' statement. I checked the number of if's versus endif just to check this wasn't a duplicate.